osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -284,12 +284,14 @@ protected override void Update()
                 // Preview user-defined background dim and blur when hovered on the visual settings panel.
                 Background.EnableUserDim.Value = true;
                 Background.BlurAmount.Value = 0;
+                Background.FadeColour(Color4.White, 250);
             }
             else
             {
                 // Returns background dim and blur to the values specified by PlayerLoader.
                 Background.EnableUserDim.Value = false;
                 Background.BlurAmount.Value = BACKGROUND_BLUR;
+                Background.FadeColour(Color4.Gray, 250);
```

Comment:
Might be a good idea to match the easing in `UserDimContainer` (at least the duration), since this dim happens much faster than the other background dim transfers right now.